.buildkite/test-pipeline.yaml

```diff
@@ -269,6 +252,57 @@ steps:
   - export VLLM_WORKER_MULTIPROC_METHOD=spawn
   - bash ./run-tests.sh -c configs/models-small.txt -t 1
 
+#####  models test  #####
+
+- label: Basic Models Test # 6min
+  source_file_dependencies:
+  - vllm/
+  - tests/models
+  commands:
+    - pip install -e ./plugins/vllm_add_dummy_model
+    - pytest -v -s models/test_oot_registration.py # it needs a clean process
+    - pytest -v -s models/*.py --ignore=models/test_oot_registration.py
+
+- label: Decoder-only Language Models Test # 1h6min
+  #mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/models/decoder_only/language
+  commands:
+    - pytest -v -s models/decoder_only/language
+
+- label: Decoder-only Audio-Language Models Test # 7min
+  #mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/models/decoder_only/audio_language
+  commands:
+    - pytest -v -s models/decoder_only/audio_language
+
+- label: Decoder-only Vision-Language Models Test # 51min
+  #mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/models/decoder_only/vision_language
+  commands:
+    - pytest -v -s models/decoder_only/vision_language
+
+- label: Embedding Language Models Test # 7min
+  #mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/models/embedding/language
+  commands:
+    - pytest -v -s models/embedding/language
+
+- label: Encoder-Decoder Language Models Test # 6min
+  #mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  - tests/models/encoder_decoder/language
+  commands:
+    - pytest -v -s models/encoder_decoder/language
```

Comment:
Oh please fuse some of the tests in one step. The issue is that booting up the test machine and container will take about 10min. so we want the test to run at least 10/15min

Reply:
Good point!